internal/feature/auth/README.md

```
@@ -283,30 +283,140 @@ auth/
 ```
 
-### All Tests
+### Run All Tests
 
 ```bash
 go test ./internal/feature/auth/... -v -race -cover
 ```
 
+### Example Test Output
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)